src/Controls/src/Core/Shapes/PathGeometry.cs

```diff
@@ -89,14 +90,25 @@ public override void AppendPath(PathF path)
 
 		void AddArc(PathF path, ArcSegment arcSegment, double density)
 		{
-			path.AddArc(
```

Comment:
Doesn't AddArc work? Why are we working with lists pf points instead of letting the OS/framework do this?

Reply:
I think we should fix it in graphics instead of working around it here.

Reply:
I agree. To include a fix soon (in this case, this problem is impacting a partner), do you mind if we merge this fix, open issue in Maui Graphics and as soon as we update Graphics with the fix we update this part?

Reply:
No problem! Help the partner out 🤗 Maybe link to an issue or the PR fix here for historical information.